.packit.yaml

```
@@ -61,6 +63,26 @@ jobs:
             - full test
           absent:
             - discuss
+    tf_extra_params:
+        test:
+            tmt:
+                name: '^(?!/plans/features/extended-unit-tests).*$'
+
+  - job: tests
+    identifier: extended-unit-tests
+    trigger: pull_request
+    targets:
+      - fedora-39
```

```suggestion
      - fedora-latest-stable
```

Let's use always the latest stable `fedora`.

---

Fixed in 190a0c2f